### ISSUE TITLE
Can't download videos from vod.tvp.pl, e.g.: https://vod.tvp.pl/live,1/tvp-1,399697

### ISSUE THREAD

**Issue by Mexit** [yt-dlp/yt-dlp#8824]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Poland

### Provide a description that is worded well enough to be understood

Can't download videos from vod.tvp.pl, e.g.: https://vod.tvp.pl/live,1/tvp-1,399697
In the browser they play correctly.
Test on two different operating systems, nightly and stable version yt-dlp

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--cookies-from-browser', 'firefox', 'https://vod.tvp.pl/live,1/tvp-2,399698', '-vU']
Extracting cookies from firefox
[debug] Extracting cookies from: "/home/manjaro/.mozilla/firefox/lmx686bz.default-release/cookies.sqlite"
Extracted 24 cookies from firefox
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.11.16 from yt-dlp/yt-dlp [24f827875]
[debug] Python 3.11.6 (CPython x86_64 64bit) - Linux-6.6.7-1-MANJARO-x86_64-with-glibc2.38 (OpenSSL 3.1.4 24 Oct 2023, glibc 2.38)
[debug] exe versions: rtmpdump 2.4
[debug] Optional libraries: certifi-2023.11.17, requests-2.31.0, secretstorage-3.3.3, sqlite3-3.44.2, urllib3-1.26.18
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests
[debug] Loaded 1901 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2023.11.16 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2023.11.16 from yt-dlp/yt-dlp)
[tvp:vod] Extracting URL: https://vod.tvp.pl/live,1/tvp-2,399698
[tvp:vod] 399698: Downloading JSON metadata
ERROR: [tvp:vod] 399698: Woronicza said: ITEM_NOT_AVAILABLE (HTTP 404); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/tvp.py", line 568, in _real_extract
    info_dict = self._parse_video(self._call_api(f'vods/{video_id}', video_id), with_url=False)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/tvp.py", line 493, in _call_api
    raise ExtractorError(f'Woronicza said: {document.get("code")} (HTTP {urlh.status})')
```

**Comment by dirkf** [comment]
Related: https://github.com/ytdl-org/youtube-dl/issues/30060

**Comment by selfisekai** [comment]
as a workaround before #8860 is merged, links to https://stream.tvp.pl/ work correctly

**Comment by selfisekai** [comment]
alternatively, try by their external id, over TVPEmbedIE:

TVP 1 - `tvp:51689486`
TVP 2 - `tvp:51696811`
TVP INFO - `tvp:51696820`
TVP KULTURA - `tvp:51696822`
TVP KOBIETA - `tvp:53795158`
TVP SPORT - `tvp:51696827`
TVP HISTORIA - `tvp:51696819`
TVP ABC - `tvp:51696812`
TVP DOKUMENT - `tvp:53795159`
TVP NAUKA - `tvp:71345993`
TVP POLONIA - `tvp:51696824`
TVP ROZRYWKA - `tvp:51696825`
TVP HISTORIA 2 - `tvp:57181934`
ALFA TVP - `tvp:65184034`
TVP ABC 2 - `tvp:57181933`
TVP WILNO - `tvp:74979895`
TVP KULTURA 2 - `tvp:56275381`
BELSAT - `tvp:64914904`
UA1 - `tvp:58758689`
TVP WORLD - `tvp:57181932`
TVP 3 BIAŁYSTOK - `tvp:71345939`
TVP 3 BYDGOSZCZ - `tvp:71345962`
TVP 3 GDAŃSK - `tvp:71345915`
TVP 3 GORZÓW WLKP. - `tvp:71345976`
TVP 3 KATOWICE - `tvp:71345996`
TVP 3 KIELCE - `tvp:71345997`
TVP 3 KRAKÓW - `tvp:71346000`
TVP 3 LUBLIN - `tvp:71346005`
TVP 3 ŁÓDŹ - `tvp:71345979`
TVP 3 OLSZTYN - `tvp:71346007`
TVP 3 OPOLE - `tvp:71346013`
TVP 3 POZNAŃ - `tvp:71346014`
TVP 3 RZESZÓW - `tvp:71346017`
TVP 3 SZCZECIN - `tvp:71345990`
TVP 3 WARSZAWA - `tvp:71345991`
TVP 3 WROCŁAW - `tvp:71346019`
TVP SERIALE - `tvp:72827372`
TVP HD - `tvp:72827367`

**Comment by rjolina** [comment]
> alternatively, try by their external id, over TVPEmbedIE:
> 
> TVP 1 - `tvp:51689486` TVP 2 - `tvp:51696811` TVP INFO - `tvp:51696820` TVP KULTURA - `tvp:51696822` TVP KOBIETA - `tvp:53795158` TVP SPORT - `tvp:51696827` TVP HISTORIA - `tvp:51696819` TVP ABC - `tvp:51696812` TVP DOKUMENT - `tvp:53795159` TVP NAUKA - `tvp:71345993` TVP POLONIA - `tvp:51696824` TVP ROZRYWKA - `tvp:51696825` TVP HISTORIA 2 - `tvp:57181934` ALFA TVP - `tvp:65184034` TVP ABC 2 - `tvp:57181933` TVP WILNO - `tvp:74979895` TVP KULTURA 2 - `tvp:56275381` BELSAT - `tvp:64914904` UA1 - `tvp:58758689` TVP WORLD - `tvp:57181932` TVP 3 BIAŁYSTOK - `tvp:71345939` TVP 3 BYDGOSZCZ - `tvp:71345962` TVP 3 GDAŃSK - `tvp:71345915` TVP 3 GORZÓW WLKP. - `tvp:71345976` TVP 3 KATOWICE - `tvp:71345996` TVP 3 KIELCE - `tvp:71345997` TVP 3 KRAKÓW - `tvp:71346000` TVP 3 LUBLIN - `tvp:71346005` TVP 3 ŁÓDŹ - `tvp:71345979` TVP 3 OLSZTYN - `tvp:71346007` TVP 3 OPOLE - `tvp:71346013` TVP 3 POZNAŃ - `tvp:71346014` TVP 3 RZESZÓW - `tvp:71346017` TVP 3 SZCZECIN - `tvp:71345990` TVP 3 WARSZAWA - `tvp:71345991` TVP 3 WROCŁAW - `tvp:71346019` TVP SERIALE - `tvp:72827372` TVP HD - `tvp:72827367`

maybe stupid question, but how to use this TVPEmbedIE with full link?
my yt-dlp version is 2023.12.30.232719

**Comment by selfisekai** [comment]
same as you would with a normal url, e.g.
```
$ yt-dlp tvp:51696820 -F
[tvp:embed] Extracting URL: tvp:51696820
[tvp:embed] 51696820: Downloading webpage
[tvp:embed] 51696820: Downloading m3u8 information
WARNING: [tvp:embed] Ignoring subtitle tracks found in the HLS manifest; if any subtitle tracks are missing, please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[tvp:embed] 51696820: Downloading m3u8 information
[info] Available formats for 51696820:
ID     EXT RESOLUTION FPS │   TBR PROTO │ VCODEC        VBR ACODEC     ABR
──────────────────────────────────────────────────────────────────────────
direct ts  unknown        │ 4000k https │ unknown           unknown
hls-0  mp4 768x432     25 │ 1666k m3u8  │ avc1.64001E 1374k mp4a.40.2 197k
hls-1  mp4 768x432     25 │ 1674k m3u8  │ avc1.64001E 1374k mp4a.40.2 197k
hls-2  mp4 1024x576    25 │ 2748k m3u8  │ avc1.64001F 2396k mp4a.40.2 197k
hls-3  mp4 1024x576    25 │ 2756k m3u8  │ avc1.64001F 2396k mp4a.40.2 197k
hls-4  mp4 1280x720    25 │ 4157k m3u8  │ avc1.64001F 3724k mp4a.40.2 197k
hls-5  mp4 1280x720    25 │ 4165k m3u8  │ avc1.64001F 3724k mp4a.40.2 197k
hls-6  mp4 1920x1080   25 │ 6756k m3u8  │ avc1.640028 6176k mp4a.40.2 197k
hls-7  mp4 1920x1080   25 │ 6764k m3u8  │ avc1.640028 6176k mp4a.40.2 197k
```

some integrations might work slightly different, e.g. the one in mpv requires `ytdl://tvp:51696820` to understand this properly
